src/plugins/memdump/userhook.cpp

```diff
@@ -135,8 +135,7 @@ static event_response_t usermode_hook_cb(drakvuf_t drakvuf, drakvuf_trap_info* i
 {
     hook_target_entry_t* target = (hook_target_entry_t*)info->trap->data;
 
-    // TODO check thread_id and cr3?
-    if (target->pid != info->proc_data.pid)
```

Comment:
Hmmm, based on comment maybe we can add `drakvuf_check_return_context` here, but that would require testing.

Reply:
No, this is not a callback on return address.

Reply:
Ah, you're right.